README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+```
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i stanza-create
+done
+```
+Под термином "stanza" понимается "строфа" в конфигурационном файле
+
+- Настроить Greenplum
+```
+gpconfig -c archive_mode -v on
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" /usr/local/bin/pgbackrest --stanza=seg%c archive-push %p'" --skipvalidation
+gpstop -ar
+```
+
+- Установить расширение gp_pitr 
+```
+create extension gp_pitr;
+```
+Это расширение необходимо для работы с точками восстановления.
+
+- Проверить, что резервное копирование и архивирование логов предзаписи правильно настроено
+```
+for i in -1 0 1 2
+do 
+	PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i check
+done
+```
+Команда не должна выдать сообщений об ошибке
+
+# 4. Резервное копирование кластера Greenplum
+
+4.1 Сохранить файлы с первичных сегментов и координатора
+```
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i backup
+done
+```
+Если потребуется остановить прерванный бэкап, то это можно сделать командой
+```
+select pg_stop_backup() from gp_dist_random('gp_id');
+```
+
+4.2 Создать именованную точку восстановления
+```
+select gp_create_restore_point('backup1');
+```
+Запись о точке восстановления будет помещена в лог предзаписи.
+
+4.3 Отправить в архив файлы, хранящие созданную точку восстановления.
+
+Отправка осуществляется через переключение логов предзаписи координатора и сегментов на новые файлы.
+```
+select gp_switch_wal();
+```
+
+# 5. Восстановление кластера Greenplum
+
+- Выключить кластер и удалить содержимое каталогов всех компонентов кластера
+```
+gpstop -a
+rm -rf /tmp/gpdb/qddir/demoDataDir-1/* /tmp/gpdb/dbfast1/demoDataDir0/* /tmp/gpdb/dbfast2/demoDataDir1/* /tmp/gpdb/dbfast3/demoDataDir2/* /tmp/gpdb/dbfast_mirror1/demoDataDir0/* /tmp/gpdb/dbfast_mirror2/demoDataDir1/* /tmp/gpdb/dbfast_mirror3/demoDataDir2/* /tmp/gpdb/standby/*
+```
+
+- Восстановить из резервной копии содержимое каталогов координатора и первичных сегментов
+
+Имя точки восстановления из пункта 4.2 передается в параметре `--target`.
+```
+for i in -1 0 1 2
+do 
+    pgbackrest --stanza=seg$i --type=name --target=backup1 restore
+done
+```
+
+- Запустить только координатор
+```
+gpstart -am
+```
+
+- Удалить резервный координатор из кластера
+```
+gpinitstandby -ar
+```
+
+- Отметить зеркала сегментов как недоступные
+```
+PGOPTIONS="-c gp_session_role=utility" psql postgres -c "
+set allow_system_table_mods to true;
+update gp_segment_configuration 
+ set status = case when role='m' then 'd' else status end, 
+     mode = 'n'
+ where content >= 0;"
+```
+
+- Перезапустить кластер в обычном режиме
+```
+gpstop -ar
+```
+
+- Восстановить зеркала по первичным сегментам
+```
+gprecoverseg -aF
+```
+
+- Восстановить резервный координатор
+```
+gpinitstandby -as $HOSTNAME -S /tmp/gpdb/standby -P 6001
+```
```

Comment:
At the end it would be good to place some command or command sequence that will allow the user to validate that recovery is fully done.